packages/app/components/Queue/Student/StudentQueue.tsx

```diff
@@ -273,6 +274,26 @@ export default function StudentQueue({
     ]
   );
 
+  useHotkeys(
+    "shift+e",
+    () => {
+      if (studentQuestion) {
+        openEditModal();
+      }
+    },
+    [studentQuestion]
+  );
+
+  useHotkeys(
+    "shift+n",
+    () => {
+      if (!studentQuestion && queue?.allowQuestions && !queue?.isDisabled) {
+        joinQueueOpenModal(false).then((res) => setShowJoinPopconfirm(!res));
```

Comment:
note: i tihnk this was mentioned at some point a long time ago, but this will console log a `400` error on the question endpoint in the backend. this also occurs when i manually click the button.